app/controllers/api/registrations_controller.rb

```diff
@@ -2,21 +2,27 @@
 module Api
   class RegistrationsController < ApplicationController
     def register_user
+      send_email && return if Rails.env.development?
```

Comment:
Skip all the recaptcha verification if in development environment.

Reply:
Oh yes that's such a good suggestion! Rubocop wants me to do it on one line but I agree that `return send_email` is a lot clearer.